c7n/resources/s3.py

```diff
@@ -1389,22 +1390,22 @@ class FilterPublicBlock(Filter):
                 region: us-east-1
                 filters:
                   - type: check-public-block
-                    scope: Any
-                    enabled: False
+                  # BlockPublicAcls: true <-- optional, if none provided sets all to true by default
```

Comment:
sets all to false actually

Reply:
Thanks, I admittedly copied this from the set action and should have been more careful with rereading. I'll be better about this in the future.  Resolved in b0b96c4